src/utils/helpers.ts

```diff
@@ -109,3 +109,7 @@ export function getSpecficCvssScore(scores: any[], issuer: string, version: stri
 export function getRhCvss3(scores: any[]) {
   return getSpecficCvssScore(scores, 'RH', 'V3');
 }
+
+export function capitalizeFirstLetter(string: string) {
```

Comment:
I think `capitalize` already means `first letter` so maybe the name can be changed?
```suggestion
export function capitalize(string: string) {
```